### DISCUSSION TITLE
Usage of Irate and increase function in kepler Dashboard

### DISCUSSION THREAD

**Discussion by MahimaK20** [sustainable-computing-io/kepler#1076] [General]
Hello All,

The kepler dashboard is using irate and increase functions for the power consumption in the cluster.
I have a query on why are we using irate function for calculating **Total Power Consumption (W) in Namespace** and increase function for **Total Power Consumption (kWh per day) in Namespace** ?
What is the significance of using two different functions? Also, can we use rate function instead of irate and increase?

**Comment by rootfs** [comment]
I believe this question has been raised before. The `irate` works with volatile data better than `rate`, but I think this is more of an implementation choice

**Comment by marceloamaral** [comment]
In essence, `rate()` shows the average change over a period, while `irate()` reveals how fast something is changing. The smoothing effect with `rate()` becomes more apparent when looking at a longer time window. As we want to show immediate changes (power fluctuation), we use `irate()`.
